src/muse/constraints.py

```diff
@@ -530,7 +530,7 @@ def minimum_service(
         .drop_vars("technology")
     )
     capacity = convert_timeslice(
-        techs.fixed_outputs * techs.utilization_factor * techs.minimum_service_factor,
+        techs.fixed_outputs * techs.minimum_service_factor,
```

Comment:
Where does the division by the maximum output happen?

Reply:
It doesn't: the minimum service factor is the result of that division, that is why it is a fraction! `fixed_outputs` would be the absolute maximum and you multiply it by `utilization_factor` or the `minimum_service_factor` to recover the upper and lower limits of the production.

Reply:
Oh I seeeee